scripts/apps/archive/directives/MediaPreview.js

```diff
@@ -2,7 +2,7 @@ MediaPreview.$inject = ['api', '$rootScope', 'desks', 'superdesk'];
 
 export function MediaPreview(api, $rootScope, desks, superdesk) {
     return {
-        templateUrl: 'scripts/apps/archive/views/preview.html',
+        template: require('../views/preview.html'),
```

Comment:
Why that is needed, doesn't template re-render on its own?

Reply:
it is to avoid using any cached version